locales/fr-FR/messages.po

```diff
@@ -1814,3 +1814,5 @@ msgstr "Votre item a été enregistré."
 msgid "ZIP archive"
 msgstr "Archive ZIP"
 
+msgid "Common Interactions"
+msgstr "Interactions communes"
```

Comment:
This file again doesn't look like it was generated, @habemuscode.
Please follow the steps as they were discussed yesterday.